src/components/user/repo/db/trashbin.schema.js

```diff
@@ -12,6 +12,7 @@ const schema = new Schema(
 		},
```

Comment:
Please do not merge this into feature/SC-7555-DeleteUserTrashbin it can develop without connections to it.